min_intensity: 200
max_intensity: 255
search_radius: 1.0 #tunable parameter for searching nearby points that are parts of uav, but without the tape - uavs shouldnt be closer to obstacle than this parameter - could filter obstacle 
max_distance_from_seed: 1.0 # maximum diameter of the uav - not all points on the uav are reflecitve
max_removed_points: 200 #[-] for one uav
ouster: false #for simulation using ouster, false if using livox on real drone - different raw data 

time_keep: 5.0
load_gt_uav_positions: false 

